components/style/properties/longhand/inherited_text.mako.rs

```diff
@@ -1265,3 +1265,11 @@ ${helpers.single_keyword("text-rendering",
                          "auto optimizespeed optimizelegibility geometricprecision",
                          animation_value_type="none",
                          spec="https://www.w3.org/TR/SVG11/painting.html#TextRenderingProperty")}
+
+${helpers.single_keyword("-moz-control-character-visibility",
+                         "hidden visible",
```

Comment:
This is actually wrong for now. The initial value of `-moz-control-character-visibility` is controlled by pref "layout.css.control-characters.visible", and can be get via `nsCSSParser::ControlCharVisibilityDefault()`. `hidden` is the default value for release versions, but we use `visible` for nightly and devedition.

Is supporting that complicated? If not, probably we should do so... I'm not sure what to do with this otherwise...